### DISCUSSION TITLE
How to make nue work with deno?

### DISCUSSION THREAD

**Discussion by ArnavK-09** [nuejs/nue#224] [Q&A]
# ?

**Comment by tipiirai** [comment]
Hey. The current focus is on CSS and then on templates. Deployments coming after those two. At least that's my own prioritization. Here's the full roadmap: https://nuejs.org/blog/perfect-web-framework/

**Reply by ArnavK-09** [reply]
> Hey. The current focus is on CSS and then on templates. Deployments coming after those two. At least that's my own prioritization. Here's the full roadmap: https://nuejs.org/blog/perfect-web-framework/

I'm asking for correct way to create nue project in deno , as it isn't working in conventional way I tried!

Ps - I want to host site on https://deno.com/deploy
